server/gfmd/db_journal_apply.c

```diff
@@ -1040,6 +1097,10 @@ const struct db_ops db_journal_apply_ops = {
 	NULL,
 
 	db_journal_apply_fsngroup_modify,
+
```

Comment:
move the user_auth related functions to just after the user related functions,
to match the modified order of struct db_ops.

Reply:
commit [429eb3d](https://github.com/oss-tsukuba/gfarm/pull/1141/commits/429eb3daa151165a09117234d565149a2a1ac4e6) has not moved the user_auth related functions to just after the user related functions yet.
Could you move them?